### ISSUE TITLE
The visual presentation of the Components/Images don't have a contrast ratio of at least 3:1 against adjacent color, at Navigation screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#855]
**Describe the bug**
The visual presentation of the Components/Images don't have a contrast ratio of at least 3:1 against adjacent color.

**To Reproduce**
1. Install "Color Contrast Analyzer" extension in your browser.
2. Go to Navigation screen by selecting any page on a site.
3. Click "Extension" icon given in the chrome browser, in top right. It will open Extension Pop Up.
4. Click "Color Contrast Analyzer", option in pop up. It will open another pop up.
5. Click "Capture Visible Content" from the pop up.
6. Observe that the folder image present before "Assets", "Design" etc is not highlighted by the analyzer. Also, the "Add Section" image at left side at Level2 is also not highlighted. So, that means they don't have the desired contrast ratio.

**Expected behavior**
The visual presentation of the Components/Images should have a contrast ratio of at least 3:1 against adjacent color.

**Additional context**
Navigate all the options available for similar issue and fix.
